assets/templates/partials/census/collapsible.tmpl

```diff
@@ -1,7 +1,7 @@
 <div id="collapsible" class="ons-collapsible ons-js-collapsible" data-btn-close="{{ localise "HideThis" .Language 1 }}">
     <div class="ons-collapsible__heading ons-js-collapsible-heading">
         <div class="ons-collapsible__controls">
-            <h2 class="ons-collapsible__title">{{ .Title }}</h2>
+            <h3 class="ons-collapsible__title">{{ localise "VariablesExplanation" .Language 2 }}&#63;</h3>
             <span class="ons-collapsible__icon">
```

Comment:
I think this needs to be a `4` if it is for `other` instead of a `two` since there's no translation for `2`? So `{{ localise "VariablesExplanation" .Language 4 }}`